plugins/woocommerce/src/Admin/Features/LaunchYourStore.php

```diff
@@ -93,4 +96,38 @@ public function preload_settings( $settings ) {
 
 		return $settings;
 	}
+
+	/**
+	 * Add 'coming soon' banner on the frontend when the following conditions met.
+	 *
+	 * - User must be either an admin or store editor (must be logged in).
+	 * - 'woocommerce_coming_soon' option value must be 'yes'
+	 */
+	public function maybe_add_coming_soon_banner_on_frontend() {
+		// User must be an admin or editor.
+		// phpcs:ignore
+		if ( ! current_user_can( 'editor' ) && ! current_user_can( 'administrator' ) ) {
```

Comment:
Could we please change editor to the [Shop Manager role](https://woo.com/document/roles-capabilities/)? Apologies, I had it wrong in the issue before (fixed it now).

Reply:
Updated in [Check for shop_manager](https://github.com/woocommerce/woocommerce/pull/46096/commits/6d5e8bea3e23b293e82c82b7179caee294987335)